src/XrdCeph/XrdCephOss.cc

```diff
@@ -156,6 +181,18 @@ int XrdCephOss::Configure(const char *configfn, XrdSysError &Eroute) {
            return 1;
          }
        }
+
+       if (!strcmp(var, "ceph.poolnames")) {
```

Comment:
The name "poolnames" suggests something that defines all possible pools (for any operations).  This option is specific for the space reporting, so I wonder if there's a better label?
spaceinfopools ?

Reply:
Now using "ceph.reportingools"